public/web.config

```diff
@@ -0,0 +1,28 @@
+<!--
```

Comment:
what is this for? email?

Reply:
Looks like it rewrites links within emails. This was something that was auto-generated with make:mail I believe

Reply:
cool just wanted to make sure it wasn't like a windows vs mac thing.